### PR TITLE
fix(tab): Add `active` className

### PR DESCRIPTION
**Pull request by mucahit**
### Description
- Added a className to active `TabHeaderItem`

### CODE REVIEW

File: src/tab/header/item/TabHeaderItem.tsx [hunk 1]
```diff
@@ -1,17 +1,19 @@
...
   onClick: (index: number) => void;
+  isActive: boolean;
 };
 
-function TabHeaderItem({children, icon, index, onClick}: TabHeaderItemProps) {
+function TabHeaderItem({children, icon, index, onClick, isActive}: TabHeaderItemProps) {
   return (
     <Button
-      customClassName={"tab-header-item"}
+      customClassName={classNames("tab-header-item", {active: isActive})}
```
Comment (burakguneli) [inline src/tab/header/item/TabHeaderItem.tsx:16]: Shouldn't you wrap `active` with quotation marks? Like: `{"active": isActive}`

**Review by burakguneli — CHANGES_REQUESTED** [review]

**Review by jamcry — APPROVED** [review]

File: src/tab/header/item/TabHeaderItem.tsx (hunk 1 above)
Comment (jamcry) [reply src/tab/header/item/TabHeaderItem.tsx:16]: I think our prettier rules automatically removes `"`s from keys if they don't include special characters (such as `-`)

Comment (edizcelik) [reply src/tab/header/item/TabHeaderItem.tsx:16]: Agreed with other comments but let's use BEM convention here
```suggestion
      customClassName={classNames("tab-header-item", {"tab-header-item--is-active": isActive})}
```

Comment (mucahit) [reply src/tab/header/item/TabHeaderItem.tsx]: Done 👍

**Review by bilgeyucel — APPROVED** [review]

**Pull request merged**